src/script/script.js

```diff
@@ -0,0 +1,402 @@
+          { className: "KeyB", ru: { down: "и", up: "И", caps: "И", shiftcaps: "и" }, en: { down: "b", up: "B", caps: "B", shiftcaps: "b" }, symbol: true },
+          { className: "KeyN", ru: { down: "т", up: "Т", caps: "Т", shiftcaps: "т" }, en: { down: "n", up: "N", caps: "N", shiftcaps: "n" }, symbol: true },
+          { className: "KeyM", ru: { down: "ь", up: "Ь", caps: "Ь", shiftcaps: "ь" }, en: { down: "m", up: "M", caps: "M", shiftcaps: "m" }, symbol: true },
+          { className: "Comma", ru: { down: "б", up: "Б", caps: "Б", shiftcaps: "б" }, en: { down: ",", up: "<", caps: ",", shiftcaps: "<" }, symbol: true },
+          { className: "Period", ru: { down: "ю", up: "Ю", caps: "Ю", shiftcaps: "ю" }, en: { down: ".", up: ">", caps: ".", shiftcaps: ">" }, symbol: true },
+          { className: "Slash", ru: { down: ".", up: ",", caps: ".", shiftcaps: "," }, en: { down: "/", up: "?", caps: "/", shiftcaps: "?" }, symbol: true },
+          { className: "ArrowUp", value: "▲", symbol: false },
+          { className: "ShiftRight", value: "Shift", symbol: false },
+        ],
+        [
+          { className: "ControlLeft", value: "Ctrl", symbol: false },
+          { className: "MetaLeft", value: "Win", symbol: false },
+          { className: "AltLeft", value: "Alt", symbol: false },
+          { className: "Space", value: "", symbol: false },
+          { className: "AltRight", value: "Alt", symbol: false },
+          { className: "ContextMenu", value: "", symbol: false },
+          { className: "ArrowLeft", value: "◄", symbol: false },
+          { className: "ArrowDown", value: "▼", symbol: false },
+          { className: "ArrowRight", value: "►", symbol: false },
+          { className: "ControlRight", value: "Ctrl", symbol: false }
+        ]
+      ];
+  }
+
+  setRootElement() {
+    this.rootElement = document.createElement('div');
+    this.rootElement.classList.add('virtual-keyboard-wrapper');
+    document.body.prepend(this.rootElement);
+  }
+  ///////////////////////////////////////////////////////////////
+  generateHeader() {
+    if (!this.rootElement) this.setRootElement();
+    this.header = document.createElement('div');
+    this.header.classList.add('header-wrapper');
+    this.header.innerHTML = `
+      <h1 class="header">RS-School task Virtual Keyboard</h1>
+      <p class="description">Клавиатура создана в операционной системе Windows</p>
+      <p class="description">Переключения языка клавиатурой: левыe ctrl + alt</p>
+      <p class="description">Переключение языка мышкой: кнопка с иконкой глобуса</p>`;
+    this.rootElement.append(this.header);
+  }
+
+  generateTextArea() {
+    if (!this.rootElement) this.setRootElement();
+    let textArea = document.createElement('div');
+    textArea = document.createElement('div');
+    textArea.classList.add('textarea-wrapper');
+    textArea.innerHTML = `
+      <textarea class="textarea" id="textarea" rows="5" cols="50" autofocus></textarea>`;
```

Comment:
You could impelement it like this:
let textArea = document.createElement('textarea') without unnecessary tags

Reply:
```suggestion
    let wrapper = document.createElement('div');
    wrapper.classList.add('textarea-wrapper');
    let textArea = document.createElement('textarea');
    textArea.classList.add('textarea');
    textArea.setAttribute('id', 'textarea');
    textArea.setAttribute('autofocus', true);
    wrapper.append(textArea);
    this.rootElement.append(wrapper);
    this.textArea = textArea;
    this.textArea.value = '';
```
Ok, I have replaced on this code

Reply:
refactored